wiki/beginners.md

```diff
@@ -0,0 +1,450 @@
+repository](https://github.com/remotestorage/modules), under
+[`dist/`](https://github.com/remotestorage/modules/dist). Their
+documentation can be found in the [modules
+docs](https://remotestorage.github.io/modules). *Beware that most common
+data modules are still under development.*
+
+If there's no data module that fits your data, you can either extend one
+or write your own — and hopefully propose a new common module.
+
+##### Example
+
+This example uses the [bookmarks
+module](https://github.com/remotestorage/modules/blob/master/dist/bookmarks.js).
+You may find helpful to take a look at the [bookmarks module
+docs](http://remotestorage.github.io/modules/files/bookmarks-js.html).
+
+First, we import `remotestorage.js` and our chosen module
+`remotestorage-bookmarks.js`.
+
+``` html5
+<!-- Require remotestorage.js -->
+<script src="remotestorage.js"></script>
+
+<!-- Require the bookmarks module -->
+<script src="remotestorage-bookmarks.js"></script>
+```
+
+Then, we can store a bookmark.
+
+``` javascript
+// Store archive-bookmark
+
+remoteStorage.bookmarks.archive.store({
+  url: 'http://jsunconf.eu',
+  title: 'JS unconference in Hamburg',
+  tags: ['javascript', 'conference']
+}).then(
+  function(bookmark){
+    console.log(bookmark);
+  },
+  function(err){
+    console.log('Error', err)
+  },
+);
+```
+
+Later on, we can request all archive-bookmarks:
+
+``` javascript
+// Get all archive-bookmarks
+
+remoteStorage.bookmarks.archive.getAll().then(
+  function(bookmarks){
+    console.log('Got bookmarks: ', bookmarks);
+  },
+  function(){
+    console.log('Something wrong is not right!')
+  },
+);
+```
+
+#### Writing your own data module
+
+If none of the available fit your data, you can define your own data
+module.
+
+##### Choosing a category (or multiple)
+
+remoteStorage data is stored in directories – also known as "categories"
+– for which your app requests access (either read-only or full). Data
+modules use root directories, i.e. right below `/`. Please choose the
+category carefully, regarding the category name to it's content and it's
+relation to other data modules.
+
+E.g. For a bookmark managing app, `/bookmarks` would be a GOOD choice of
+category, while `/stuff` or `/app` would be BAD.
+
+##### Define a module
+
+The module name defines which root folder it will use, e.g. the
+following module named `bookmarks` uses the folder `/bookmarks`. This
+does NOT give read/write permissions to the end-script, which should do
+it through
+[remoteStorage.access.claim](https://remotestorage.github.io/remotestorage.js/files/access-js.html#RemoteStorage.Access.claim).
+
+You can find further information on the docs on
+[RemoteStorage.defineModule](https://remotestorage.github.io/remotestorage.js/files/modules-js.html).
+
+There are two helpers, which are instances of
+[RemoteStorage.BaseClient](https://remotestorage.github.io/remotestorage.js/files/baseclient-js.html):
+
+:;privateClient
```

Comment:
The next eleven lines are broken somehow. I don't know what's intended here, so I cannot comment on how fixing this.